manim/mobject/vector_field.py

```diff
@@ -1048,5 +1080,9 @@ def finish_updater_cycle(line, alpha):
                 )
         return AnimationGroup(*animations)
 
+    def fit_to_coordinate_system(self, coordinate_system: CoordinateSystem):
+        for line in self.submobjects:
```

Comment:
Did not check locally, just out of curiosity: is this loop really required? Wouldn't the `apply_function` strategy from the parent class also work correctly?

Reply:
It would work, i just left that one in because i tried different implementations as always.